Dockerfile

```diff
@@ -117,6 +126,15 @@ RUN rm -f /usr/bin/python /usr/bin/pip \
  && az --version \
  && docker-compose --version
 
+#-- .NET SDK
+RUN wget https://dot.net/v1/dotnet-install.sh \
+ && chmod +x ./dotnet-install.sh \
+ && ./dotnet-install.sh -c ${DOT_NET_SDK_VERSION} \
+ && echo 'export PATH="$HOME/.dotnet:$PATH"' >> ~/.bash_profile \
+ && source ~/.bash_profile \
```

Comment:
This may not work as you expect. `Build-Bus` is built using the root user, then towards the bottom the user is changed to `node`. In this instance, you would be appending the the `/users/root/.bash_profile` or similar

Reply:
A good way to validate is to add `RUN dotnet --version` to the bottom of the file (near the Lein one). This will ensure it's configured for the `node` user.